ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/Contracts.kt

```diff
@@ -26,3 +27,29 @@ interface Contract {
  */
 @CordaSerializable
 interface Command
+
+/**
+ * Defines a verifiable command; that is, a command that implements its own contract verification logic.
+ */
+interface VerifiableCommand : Command {
```

Comment:
If we're going to have a few types of `Command`s, then I think a separate file/files and possibly a `commands` package if we split into multiple files.

Reply:
Happy to split. I actually had everything in it's own file (which I prefer), but the list of files was becoming a bit massive.

Reply:
This is a 'new' feature we will have to think a bit in terms of if this is going to give us problems with evolution. The rule for Contract is clear in that we identify what contracts are used on all inputs+outputs and invoke them all. If the input comes from an older version of the contract we require that the new sandbox environment has an equivalent Contract that we can map to, otherwise this is an invalid transaction. I guess we can do the same for commands, but it may be more complex to remember as a coder.

Reply:
Again my view is that we shouldn't be putting any new design elements into the first few PR's on UTXO ledger, but rather hold off till we are in a position to see the consequences of adding them in